src/data/cassandra/Schema.hpp

```diff
@@ -268,6 +268,21 @@ class Schema {
             settingsProvider_.get().getTtl()
         ));
 
+        statements.emplace_back(fmt::format(
+            R"(
+           CREATE TABLE IF NOT EXISTS {}
+                  ( 
+                    mpt_id blob,
+                    holder  blob,
```

Comment:
Formatting is not the same as with other existing create statements.. it's very minor but will not pass review 🔢

Reply:
Im already on clang format version 17. And it gives me this format when I run `clang-format -i ` on this file

Reply:
Clang-format does nothing to user-provided strings. I'm referring to the formatting of the CQL inside the string.